docs/4.0/aws_oss_guide.md

```diff
@@ -0,0 +1,342 @@
+# Running Teleport on AWS
+
+We've created this guide to give customers a high level overview of how to use Teleport
+on Amazon Web Services (AWS). This guide starts with a high level introduction and 
+with a deeper dive into how to setup Teleport and run Teleport in production. 
+We have split this guide into. 
+
+- [Teleport on AWS FAQ](#teleport-on-aws-faq)
+- [Setting up Teleport OSS on AWS](#setting-up-teleport-oss-on-aws)
+- [Setting up Teleport Enterprise on AWS](#running-teleport-enterprise-on-aws)
+- [Teleport AWS Tips & Tricks](#teleport-aws-tips-tricks)
+
+### Teleport on AWS FAQ
+**Why would you want to use Teleport with AWS?**
+
+At some point you'll want to log into the system using SSH
+to help test, debug and troubleshoot a problem box. For EC2, AWS recommends creating 
+['Key Pairs'](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) 
+and has a range of [other tips for securing EC2 instances](https://aws.amazon.com/articles/tips-for-securing-your-ec2-instance/).
+
+There is a range of limits with this setup. 
+
+1. As your organization grows keeping track of end users public/private keys becomes
```

Comment:
`organization grows, keeping track`

`end users'`

Reply:
I fixed the top one, but what do you mean by `end users' `

Reply:
It's the way you write a plural where the last character of the word you're pluralising is an `s`

So rather than writing `end users's public keys`, you write `end users' public keys` - as in `the public keys belonging to the end users`

So the edited sentence becomes :`As your organization grows, keeping track of end users' public/private keys becomes...`